api/handler/triggers.go

```diff
@@ -74,6 +84,10 @@ func createTrigger(writer http.ResponseWriter, request *http.Request) {
 		return
 	}
 
+	if problems != nil {
+		response.CheckResult.Targets = problems
```

Comment:
hmm, in writeErrorSaveResponse already init?

Reply:
`writeErrorSaveResponse` is only for error (red color functions), and there is `return` [on 47th line](https://github.com/moira-alert/moira/pull/817/files#diff-a99418ecdfd4ca5c347514b729fa53b494213b93e43328a6a753a32238d7abe1R47). Here is on 88th line writing problems to response for warn (yellow color functions), so it saves trigger but writes problems to response body

Reply:
hm, i think after all we should move it into save func....